recipes-core/packagegroups/packagegroup-ni-snac.bb

```diff
@@ -0,0 +1,14 @@
+SUMMARY = "Open source package dependencies for the NILRT SNAC configuration."
+LICENSE = "MIT"
+
+
+inherit packagegroup
+
+
+RDEPENDS:${PN} = ""
+
+RDEPENDS:${PN}:append = "\
```

Comment:
Why list these in `RDEPENDS:${PN}:append` and not in `RDEPENDS:${PN}`?

Reply:
I guess force of habit - since these packagegroups normally have many subsections. But I guess it doesn't make sense in this case. Will fix.

Reply:
Fixed.